mapflow/mapflow.py

```diff
@@ -2670,18 +2685,15 @@ def unload(self) -> None:
     def read_mapflow_token(self) -> None:
         """Compose and memorize the user's credentils as Basic Auth."""
         if self.use_oauth:
-            print("Using oauth")
```

Comment:
Forgot to update my dev after previous merging, but I made it the same manualy. Hopefully, I didn't break anything, still am not a confident git user)))

Reply:
it's better to merge the dev back here, but in this case no big deal